src/components/login.js

```diff
@@ -136,19 +144,24 @@ const Login = (props) => {
               )}
             </Form.Item>
             <Form.Item {...tailFormItemLayout}>
-              <Button type='primary' htmlType='submit'>
```

Comment:
Getting a massive red warning, just need you change htmlType to htmltype. Besides that though it looks great and ready to merge after someone else checks it out and approves it!
![change htmlType to htmltype](https://user-images.githubusercontent.com/51389138/84334485-511b6900-ab47-11ea-953a-8a4210982ee4.PNG)